data-access-analysis-tools/index-portal-intro.rst

```diff
@@ -119,9 +121,59 @@ Note that corresponding plot point for the selected row in the table is differen
 
     An updated results view in which the xy plot uses the magnitude columns.
 
-See also the :ref:`Examples-DP0-1-Notebooks` for additional demonstrations of how to use the Portal's Single Table Query.
+See also the for additional demonstrations of how to use the Portal's Single Table Query.:ref:`Examples-DP0-1-Notebooks`
 
 ADQL Queries
 ============
 
- 
+**2. Select Query Type**: Select 'ADQL' to query via the ADQL interface. ADQL, `Astronomical Data Query Language <https://www.ivoa.net/documents/ADQL/>`_, is the language used by  the `IVOA <https://ivoa.net>`_ to represent astronomy queries posted to Virtual Observatory (VO) services, such as the Rubin LSST TAP service. ADQL is based on the Structured Query Language (SQL).
+
+**3. Advanced ADQL**: When ADQL is selected as the query type, the interface in  **3.** changes to provide a free-form block into which ADQL queries can be entered directly. The query excuted in the :ref:`single-table-queries` example above can be expressed in ADQL as follows:
+
+.. code-block:: SQL
+
+   SELECT ra, dec, mag_g, mag_i
+   FROM dp01_dc2_catalogs.object
+   WHERE CONTAINS(
+   POINT('ICRS', ra, dec),
+   CIRCLE('ICRS', 61.863, -35.79, 0.05555555555555555))=1
+   AND (mag_g <24 AND mag_i <24)
+
+Type the above query into the ADQL Query block and click on the 'Search' button in the bottom left corner to execute. You should set the row limit to be a small number, such as 10,  when first testing queries. The search results will populate the same **Results View**, as shown above using the Single Table Query interface. A total of 205 records should be returned, which you can interact with in the sam manner as outlined in :ref:`single-table-queries`.
+
+**Joining with another tables**
+ It is often desireable to access data stored in more than just one table. We do this using a JOIN clause to combine rows from two or more tables. Here we will join the data in the object table with the data in the truth table to compare the results of the processing with the input truth information. The two tables are joined by matching the objectIds across two catalogs.
+
+.. code-block:: SQL
+
+   SELECT obj.ra, obj.dec, obj.mag_g, obj.mag_i, truth.ra, truth.dec, truth.mag_g, truth.mag_i
```

Comment:
This line cuts off when displayed on my browser. May have to break it into two lines.